x-pack/plugins/apm/server/routes/errors/route.ts

```diff
@@ -18,9 +18,13 @@ import {
   comparisonRangeRt,
 } from '../default_api_types';
 import { createApmServerRouteRepository } from '../apm_routes/create_apm_server_route_repository';
+import { getServiceErrorGroupMainStatistics } from './get_error_groups/get_error_group_main_statistics';
+import { getServiceErrorGroupPeriods } from './get_error_groups/get_error_group_detailed_statistics';
```

Comment:
nit:

```suggestion
import { getErrorGroupMainStatistics } from './get_error_groups/get_error_group_main_statistics';
import { getErrorGroupPeriods } from './get_error_groups/get_error_group_detailed_statistics';
```

Reply:
For the detailed statistics there are two different functions there, I'm going to name one `getErrorGroupPeriods` and the other one `getErrorGroupDetailedStatistics`, the one we need to import is the '...Periods'